hub/core/version_control/commit_diff.py

```diff
@@ -0,0 +1,75 @@
+from typing import Set
+from hub.core.storage.cachable import Cachable
+
+
+class CommitDiff(Cachable):
+    """Stores set of diffs stored for a particular tensor in a commit."""
+
+    def __init__(self, created=False) -> None:
```

Comment:
What is a reasoning behind storing it ?

Reply:
It's stored so we can track the differences between commits without actually looking at the data inside.